js/treetest-edit.js

```diff
@@ -20,24 +20,59 @@ function bindFunctions(){
     	$('#form').submit()
 	});
 
-  $("#createNewRootNode").click(function() {
+  	$("#createNewRootNode").click(function() {
   		$('#tree').jstree().create_node("#", "New Root Item", "last");
 	});
 
 	//expand parent node when child is created
-  $("#createNewNode").click(function() {
-  	var selectedParent = $("#tree").jstree("get_selected");
-  	$('#tree').jstree().create_node(selectedParent, "New Item", "last");
-  	$("#tree").jstree("open_node", selectedParent);
+  	$("#createNewNode").click(function() {
+  		var selectedParent = $("#tree").jstree("get_selected");
+  		$('#tree').jstree().create_node(selectedParent, "New Item", "last");
+  		$("#tree").jstree("open_node", selectedParent);
 	});
 
-  $("#expandAll").click(function() {
-  	$("#tree").jstree('open_all');
+  	$("#expandAll").click(function() {
+  		$("#tree").jstree('open_all');
 	});
 
 	$('#tree').on("changed.jstree", function (e, data) {
 		$("#tree").jstree("toggle_node", data.selected);
 	});
+
+	$("#newTask").on("click", function() {
+		var newRow = document.getElementById('newTask').querySelector('template').content.cloneNode(true);
+		$("#tasks").children("tbody").append($(newRow));
+	});
+
+	$("#tasks").on("click", ".deleteTaskBtn", function () {
+        $(this).closest("tr").remove();   
+    });
+
+	$("#tasks").on("click", ".addAnswerBtn", function () {
+		var node = $("#tree").jstree("get_selected");
+		var nodes = $('#tree').jstree('get_path', node, '/').split('/');

```

Comment:
```suggestion
		var nodes = $('#tree').jstree().get_path(node);
```

This has the same effect when I try it!

![screenshot of trying different syntaxes in the browser console. Each syntax gets the same output](https://github.com/ScottLogic/kort/assets/118172583/c94feec9-b3a5-4afa-9e0c-66ad2ab0349e)